app/Main.hs

```diff
@@ -52,7 +52,7 @@ rules env = do
   create ["archive.html"] $ do
     route idRoute
     compile $ do
-      tutorials <- recentFirst =<< loadAll "tutorials/haskell/*/*.md"
+      tutorials <- recentFirst =<< loadAll "tutorials/(haskell|functional-full-stack)/*/*.md"
```

Comment:
@javcasas You can use Hakyll's `(.||.)` and make it a pattern to reuse everywhere:

```haskell
pattern :: [Pattern]
pattern =
  "tutorials/functional-full-stack/*/*.md"
    .||. "tutorials/haskell/*/*.md"
```

Alternatively, make a list of languages (or some other name) and map over each one to make a list of patterns, and then fold using `(.||.)`.

Or use Hakyll's `fromRegex` if you really want to use something besides `*`.